contrib/drivers/drivers.go

```diff
@@ -57,6 +57,12 @@ type VolumeDriver interface {
 
 	DeleteSnapshot(opt *pb.DeleteVolumeSnapshotOpts) error
 
+	CreateVolumeGroup(opt *pb.CreateVolumeGroupOpts, group *model.VolumeGroupSpec) (*model.VolumeGroupSpec, int, error)
```

Comment:
Please remove `int` in return value, you can pass the `Not Implemented` value into error

Reply:
I set Not Implemented to bool, because implement error must be separated from other error, and there is no error type judgment in go language like in Python.